stdlib/LinearAlgebra/src/cholesky.jl

```diff
@@ -106,7 +106,7 @@ Base.iterate(C::Cholesky, ::Val{:done}) = nothing
     CholeskyPivoted
 
 Matrix factorization type of the pivoted Cholesky factorization of a dense symmetric/Hermitian
-positive semi-definite matrix `A`. This is the return type of [`cholesky(_, Val(true))`](@ref),
+positive semi-definite matrix `A`. This is the return type of [`cholesky(_, ::RowMaximum)`](@ref),
```

Comment:
The current cross reference doesn't work properly and links to the cholesky-docstring from SparseArrays.jl. Is this type-based reference better or how do I specify which docstring to link to? @fredrikekre